installing/installing_sno/install-sno-installing-sno.adoc

```diff
@@ -6,16 +6,42 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+You can install OpenShift on a single node using an ISO that you generate.
+
 ifndef::openshift-origin[]
-include::modules/install-sno-generating-the-discovery-iso-with-the-assisted-installer.adoc[leveloffset=+1]
-endif::openshift-origin[]
 
-include::modules/install-sno-generating-the-discovery-iso-manually.adoc[leveloffset=+1]
+== Installing {sno} using the Assisted Installer
 
-include::modules/install-sno-installing-with-usb-media.adoc[leveloffset=+1]
+To install OpenShift on a single node, use the web-based Assisted Installer wizard to guide you through the process and manage the installation.
+
+include::modules/install-sno-generating-the-discovery-iso-with-the-assisted-installer.adoc[leveloffset=+2]
+
+include::modules/install-sno-installing-with-the-assisted-installer.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about creating a bootable USB drive, see xref:../../installing/installing_sno/install-sno-installing-sno.adoc#installing-with-usb-media_install-sno-installing-sno-with-the-assisted-installer[Creating a bootable ISO image on a USB drive].
```

Comment:
Since its mentioned in the square brackets, I suggest:

_- xref:../../installing/installing_sno/install-sno-installing-sno.adoc#installing-with-usb-media_install-sno-installing-sno-with-the-assisted-installer[Creating a bootable ISO image on a USB drive]_